src/librustc_mir/transform/qualify_min_const_fn.rs

```diff
@@ -367,13 +368,5 @@ fn check_terminator(
         TerminatorKind::Assert { cond, expected: _, msg: _, target: _, cleanup: _ } => {
             check_operand(tcx, cond, span, def_id, body)
         }
-
-        TerminatorKind::FalseUnwind { .. } if feature_allowed(tcx, def_id, sym::const_loop) => {
-            Ok(())
-        }
-
-        TerminatorKind::FalseUnwind { .. } => {
-            Err((span, "loops are not allowed in const fn".into()))
```

Comment:
Btw, if @centril wants to keep a check in here, I would suggest doing something like this instead, for a correct approach: https://github.com/rust-lang/rust/blob/71c6346aa1afa5a0d246649d4919e11311e635f2/src/librustc_mir/transform/check_consts/validation.rs#L191-L195

Reply:
I'm fine with this. :)